src/jobs/publish_s3.yml

```diff
@@ -0,0 +1,32 @@
+description: Publish files to S3
+
+executor: aws
+
+parameters:
+  public:
```

Comment:
Do we have private repos?

Reply:
This is to allow us in the future to publish private files to S3.
Since I don't want to have a default for this parameter when it appears, I thought to add this to future-proof the orb if we want to have this feature later. Of course the decision is not taken so this can be revisited.

Reply:
We ended up removing this for now. It is easy to add the option to publish to a private bucket in the future, which can default to false, thus preventing the need to go and change all the already working workflows.